src/index.html

```diff
@@ -9,10 +31,47 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
```

Comment:
![image](https://github.com/user-attachments/assets/6e3e8365-0374-4cc0-a146-fe0e9a399058)

```suggestion
      href="./styles/index.scss"
```